server/live/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Disaster Ninja by kontur.io</title>
+  <meta name="description"
+        content="Disaster Ninja is a platform for assesment of mapping needs in case of natural disaster. See what's happening, what are available map sources, explore coverage quality of OpenStreetMap, chart the area of interest, export and import as geojson.">
+  <meta http-equiv="refresh" content="0; url=/active/"/>
+  <script>
+    const hash = window.location.hash.substring(1);
```

Comment:
You can also briefly describe what that script does, for example
```suggestion
  <script>
  // This script takes url hash, transforms it to DN 2 url format and initiates reirect with those params
    const hash = window.location.hash.substring(1);
```
Or add comments according to the actions described